pkg/test/helper.go

```diff
@@ -0,0 +1,19 @@
+package test
+
+import (
+	"os"
+	"testing"
+)
+
+func EnvWrapper(envVars map[string]string, testFunc func(t *testing.T)) func(t *testing.T) {
```

Comment:
Smart og pen løsning
Den kan dog overskrive eksiterende miljøvariabler som ikke blir gjennopprettet etter testen er kjørt.

Reply:
will fix

Reply:
Enig 😻 , men lite sannsynlig at det er et issue for test-kode der man har grei kontroll på disse

Reply:
Enig med lite sansynlig, men hvis man ført får en sånn feil blir det fort vanskelig å feilsøke